releases/release-1.26/release-notes/maps/pr-110907-map.yaml

```diff
@@ -0,0 +1,6 @@
+pr: 110907
+releasenote:
+  text: |
+    'kubectl apply: Added a warning that kubectl will ignore no-namespaced resource
+    `pv & namespace` in a future release if the namespace is specified and allowlist is
```

Comment:
I think this entire thing needs a bit of re-wording.. That statement has too many breaks to read in a full flow